scripts/README.md

```diff
@@ -7,11 +7,12 @@
 | `test_examples.sh`      | This script tests whether a change affects example files bundled in the website.                                                      |
 | `check-headers-file.sh` | This script checks the headers if you are in a production environment.                                                                |
 | `diff_l10n_branches.py` | This script generates a report of outdated contents in `content/<l10n-lang>` directory by comparing two l10n team milestone branches. |
-| `hash-files.sh` | This script emits as hash for the files listed in $@ |
-| `linkchecker.py` | This a link checker for Kubernetes documentation website. |
-| `lsync.sh` | This script checks if the English version of a page has changed since a localized page has been committed. |
-| `replace-capture.sh` | This script sets K8S_WEBSITE in your env to your docs website root or rely on this script to determine it automatically |
-| `check-ctrlcode.py` | This script finds control-code(0x00-0x1f) in text files. |
+| `hash-files.sh`         | This script emits as hash for the files listed in $@                                                                                  |
+| `linkchecker.py`        | This a link checker for Kubernetes documentation website.                                                                             |
+| `lsync.sh`              | This script checks if the English version of a page has changed since a localized page has been committed.                            |
+| `replace-capture.sh`    | This script sets K8S_WEBSITE in your env to your docs website root or rely on this script to determine it automatically               |
+| `check-ctrlcode.py`     | This script finds control-code(0x00-0x1f) in text files.                                                                              |
+| `draft-translate.py`    | This script creates a draft translation for a given file.                                                                             |
```

Comment:
I'd like to add a very clear caveat that a human localizer must then check that that translation is right for the target language, possibly with a link to some advice about what to look out for (eg, tooling has localized a shortcode not on the ignore list).

Reply:
Thank you for your comments.
I have added caveat on script description in REAME.md.
I am not sure about detailed limitations of the library, so writing only 'general' shortcomings of using library is possible.